cdk/lib/cdk-stack.test.ts

```diff
@@ -5,7 +5,7 @@ import { CdkStack } from "./cdk-stack";
 describe("The tag-janitor stack", () => {
   it("matches the snapshot", () => {
     const app = new App();
-    const stack = new CdkStack(app, "tag-janitor");
+    const stack = new CdkStack(app, "tag-janitor", { app: "tag-janitor" });
```

Comment:
I wonder how often the second argument (`id`) would differ from the app name, and whether we could therefore just use it and avoid having an optional extra argument?

```typescript
    const stack = new CdkStack(app, "tag-janitor");
```